.circleci/config.yml

```diff
@@ -63,7 +63,7 @@ jobs:
           name: clean up running jobs
           command: |
             . venv/bin/activate
-            python cleanup_jobs.py
+            python cleanup_jobs.py || echo "Cleanup failed"
```

Comment:
temp change: will be removed in https://github.com/neuromation/cookiecutter-neuro-project/issues/78